.eslintrc.js

```diff
@@ -366,8 +369,12 @@ module.exports = {
         'jest/no-disabled-tests': 'warn',
         'jest/no-focused-tests': 'warn',
         'jest/no-identical-title': 'error',
+        'jest/no-mocks-import': 'off',
         'jest/valid-expect': 'error',
         'jest/expect-expect': ['error', { assertFunctionNames: ['expect', 'measurePerformance'] }],
+        'jest/valid-title': 'off',
```

Comment:
pourquoi celle-ci aussi on la désactive ?

Reply:
Je l'avais désactivé car par défaut, il y avait "should" dans les mots bannis, et pas sûr qu'on voulait suivre ça.
Je viens de modifier : je l'active mais en overridant la liste des mots bannis (on a le droit à should, du coup)